### DISCUSSION TITLE
> When I click "Open Your Nextcloud" button on containers url page, this page reopen again, and I never can go to login page.

### DISCUSSION THREAD

**Discussion by AliSabzali** [nextcloud/all-in-one#2648] [Questions] [answered]
> When I click "Open Your Nextcloud" button on containers url page, this page reopen again, and I never can go to login page.
> There is no issue on apache logs.
> it is so weird issue.
> My topology in our network: I have used cloudflare tunnel and pfsene firewall in my network.

![image](https://github.com/nextcloud/all-in-one/assets/10707957/f0bc62cd-8e7f-42b8-b567-d56fe69e5c10)

**apache log file**

![image](https://github.com/nextcloud/all-in-one/assets/10707957/fadc6844-e95d-4374-8da6-49efdadfd8dc)

_Originally posted by @AliSabzali in https://github.com/nextcloud/all-in-one/discussions/2105#discussioncomment-6052955_

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#how-to-run-nextcloud-behind-a-cloudflare-tunnel

**Reply by AliSabzali** [reply]
As you can see in the first picture, I installed it correctly according to the steps on your website and all the containers are running. So the problem is not in installation and access. Because it can be accessed through the web, not the local network, as you can see in the website address in the browser. So the problem is not in the tunnel and firewall. Rather, it is an internal problem in your product.

**Reply by szaimen** [reply]
> Rather, it is an internal problem in your product.

No it is not. Did you already followed https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things, espwcially point 2?
